data/json/overmap/overmap_special/specials.json

```diff
@@ -712,7 +712,8 @@
     "id": "Cabin_Lapin",
     "overmaps": [
       { "point": [ 0, 0, 0 ], "overmap": "cabin_lapin_north" },
-      { "point": [ 0, 0, 1 ], "overmap": "cabin_roof_lapin_north" }
+      { "point": [ 0, 0, 1 ], "overmap": "cabin_roof_lapin_north" },
+      { "point": [ 1, 1, 0 ], "overmap": "field_graboid" }
```

Comment:
So Lapin's cabin now has a neutral Graboid worm named Little Guy around it?  Should Lapin mention this?

Reply:
Not the cabin itself, it just spawns on an overmap tile nearby. It's not immediately visible. I do wish Lapin mentioned it, like I mentioned it in my last comment, I am just horrendous at writing dialogue and would like to avoid that.